src/app/resources/App.styl

```diff
@@ -90,14 +90,11 @@ html, body, .app
                 width 32px
                 height 32px
                 &.nav-btn:hover
-                    background-color #eee
-                    border-color #57585A;
+                    border-color $toolbarBorder
                 &:hover
-                    background-color #333
-                    background-color rgba(102, 102, 102, 0.90)
+                    background-color $toolbarButtonHover
                 &:active
-                    background-color #ccc
-                    background-color rgba(102, 102, 102, 0.60)
+                    background-color $toolbarButton
```

Comment:
I think that this color could be darker to create more contrast. It's hard to tell if the button is selected or not.